src/Symfony/Bundle/FrameworkBundle/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ CHANGELOG
  * Add support for assets pre-compression
  * Rename `TranslationUpdateCommand` to `TranslationExtractCommand`
  * Add JsonEncoder services and configuration
+ * Add new `framework.property_info.with_constructor_extractor` option to allow enabling or disabling the constructor extractor integration
```

Comment:
I think we must update the `UPGRADE-7.3.md` file as well, as we trigger a deprecation

Reply:
Done